### PR TITLE
[1] Start Ping360 support integration with new protocol structure

### PR DESCRIPTION
**Pull request by patrickelectric**
Needs: https://github.com/bluerobotics/ping-protocol/pull/132
This update ping-submodule to use master
Also allows device manager development
Needs Ping 3.27 firmware release.
^ https://github.com/bluerobotics/ping-firmware-private/releases/tag/3.27rc

### CODE REVIEW

**Comment by patrickelectric** [comment]
@jaxxzer @Williangalvani ^

File: src/sensor/protocoldetector.cpp [hunk 1]
```diff
@@ -147,8 +146,8 @@ bool ProtocolDetector::checkSerial(LinkConfiguration& linkConf)
 bool ProtocolDetector::checkUdp(LinkConfiguration& linkConf)
 {
     // To find a ping, we this message on a link, then wait for a reply
-    ping_msg_ping1D_empty req;
-    req.set_id(Ping1DNamespace::Firmware_version);
+    ping_message_general_request req;
```
Comment (jaxxzer) [inline src/sensor/protocoldetector.cpp]: Are we ready to make this change?

Comment (patrickelectric) [reply src/sensor/protocoldetector.cpp]: Needs 3.27 to be release first.

**Comment by jaxxzer** [comment]
@patrickelectric we should format the request according to the firmware version. We can remove the retro-support after ~6 months.

File: src/sensor/protocoldetector.cpp (hunk 1 above)
Comment (jaxxzer) [reply src/sensor/protocoldetector.cpp]: @patrickelectric we should format the request according to the firmware version. We can remove the retro-support after ~6 months.

Comment (patrickelectric) [reply src/sensor/protocoldetector.cpp]: `general_request` is ok to use in `ProtocolDetector`, the only problem that is know after Ping1D release is while using `general_request` with `profile`. This patch is already compatible with old firmware versions.

**Pull request merged**